share/completions/sfdx.fish

```diff
@@ -0,0 +1,1086 @@
+# Tab completion for sfdx (https://developer.salesforce.com/tools/sfdxcli).
+
+function __fish_sfdx_needs_command
+    set cmd (commandline -opc)
+    if [ (count $cmd) -eq 1 ]
+        return 0
+    end
+    return 1
+end
+
+function __fish_sfdx_using_command
+    set cmd (commandline -opc)
+    if [ (count $cmd) -gt 1 ]
+        if [ $argv[1] = $cmd[2] ]
+            return 0
+        end
+    end
+    return 1
+end
+
+function __fish_sfdx_using_option
```

Comment:
This can be replaced with `__fish_sfdx_using_command $argv[1]; and __fish_contains_opt $argv[2]` (without the "--" for the option)

Reply:
Thank you for reviewing again. But I think it cannot be replaced with that because these are not the same.

Reply:
Sorry, these are not the same but it's not a problem. I can do that.

Reply:
I did it!